x-pack/legacy/plugins/siem/public/mock/news.ts

```diff
@@ -0,0 +1,139 @@
 into the same experience to streamline how you secure your organization."},"link_text":null,"link_url":{"en":"https://www.elastic.co/webinars/elastic-endpoint-security-overview-security-starts-at-the-endpoint?blade=securitysolutionfeed"},"languages":null,"badge":{"en":"7.5"},"image_url":{"en":"https://static-www.elastic.co/v3/assets/bltefdd0b53724fa2ce/bltd0eb8689eafe398a/5d970ecc1970e80e85277925/illustration-endpoint-hero.png?blade=securitysolutionfeed"},"publish_on":"2019-11-17T00:00:00","expire_on":"2020-12-31T00:00:00","hash":"ec970adc85e9eede83f77e4cc6a6fea00cd7822cbe48a71dc2c5f1df10939196"},{"title":{"en":"Trying Elastic SIEM at Home?"},"description":{"en":"For small businesses and homes, having access to effective security analytics can come at a high cost of either time or money. Well, until now!"},"link_text":null,"link_url":{"en":"https://www.elastic.co/blog/elastic-siem-for-small-business-and-home-1-getting-started?blade=securitysolutionfeed"},"languages":null,"badge":{"en":"7.5"},"image_url":{"en":"https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt024c26b7636cb24f/5daf4e293a326d6df6c0e025/home-siem-blog-1-map.jpg?blade=securitysolutionfeed"},"publish_on":"2019-10-24T00:00:00","expire_on":"2020-12-31T00:00:00","hash":"aa243fd5845356a5ccd54a7a11b208ed307e0d88158873b1fcf7d1164b739bac"},{"title":{"en":"Introducing Elastic Endpoint Security"},"description":{"en":"Elastic is excited to announce the introduction of Elastic Endpoint Security, based on Elastic’s acquisition of Endgame, a pioneer and industry-recognized leader in endpoint threat prevention, detection, and response."},"link_text":null,"link_url":{"en":"https://www.elastic.co/blog/introducing-elastic-endpoint-security?blade=securitysolutionfeed"},"languages":null,"badge":{"en":"7.5"},"image_url":{"en":"https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt1f87637fb7870298/5d9fe27bf8ca980f8717f6f8/screenshot-resolver-trickbot-enrichments-showing-defender-shutdown-endgame-2-optimized.png?blade=securitysolutionfeed"},"publish_on":"2019-10-15T00:00:00","expire_on":"2020-03-01T00:00:00","hash":"3c64576c9749d33ff98726d641cdf2fb2bfde3dd9a6f99ff2573ac8d8c5b2c02"},{"title":{"en":"What is Elastic Common Schema (ECS)?"},"description":{"en":"Elastic SIEM is powered by Elastic Common Schema. With ECS, analytics content such as dashboards, rules, and machine learning jobs can be applied more broadly, searches can be crafted more narrowly, and field names are easier to remember."},"link_text":null,"link_url":{"en":"https://www.elastic.co/blog/introducing-the-elastic-common-schema?blade=securitysolutionfeed"},"languages":null,"badge":{"en":"7.0"},"image_url":{"en":"https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt71256f06dc672546/5c98d595975fd58f4d12646d/ecs-intro-dashboard-1360.jpg?blade=securitysolutionfeed"},"publish_on":"2019-02-13T00:00:00","expire_on":"2020-12-31T00:00:00","hash":"b8a0d3d21e9638bde891ab5eb32594b3d7a3daacc7f0900c6dd506d5d7b42410"}]}`;
```

Comment:
nit: maybe we could extract it to a separate json file? in case we want to modify it later

Reply:
Good call, I extracted it to a separate file and also ran it through a JSON formatter